Assets/MRTK/Core/Utilities/Gltf/Schema/GltfSampler.cs

```diff
@@ -10,27 +11,83 @@ namespace Microsoft.MixedReality.Toolkit.Utilities.Gltf.Schema
     /// https://github.com/KhronosGroup/glTF/blob/master/specification/2.0/schema/sampler.schema.json
     /// </summary>
     [Serializable]
-    public class GltfSampler : GltfChildOfRootProperty
+    public class GltfSampler : GltfChildOfRootProperty, ISerializationCallbackReceiver
     {
         /// <summary>
         /// Magnification filter.
         /// Valid values correspond to WebGL enums: `9728` (NEAREST) and `9729` (LINEAR).
         /// </summary>
-        public GltfMagnificationFilterMode magFilter = GltfMagnificationFilterMode.Linear;
+        public GltfMagnificationFilterMode MagFilter { get; set; }
+
+        [SerializeField]
+        private string magFilter = null;
 
         /// <summary>
         /// Minification filter. All valid values correspond to WebGL enums.
         /// </summary>
-        public GltfMinFilterMode GltfMinFilter = GltfMinFilterMode.NearestMipmapLinear;
+        public GltfMinFilterMode MinFilter { get; set; }
+
+        [SerializeField]
+        private string minFilter = null;
 
         /// <summary>
         /// s wrapping mode.  All valid values correspond to WebGL enums.
         /// </summary>
-        public GltfWrapMode wrapS = GltfWrapMode.Repeat;
+        public GltfWrapMode WrapS { get; set; }
+
+        [SerializeField]
+        private string wrapS = null;
 
         /// <summary>
         /// t wrapping mode.  All valid values correspond to WebGL enums.
         /// </summary>
-        public GltfWrapMode wrapT = GltfWrapMode.Repeat;
+        public GltfWrapMode WrapT { get; set; }
+
+        [SerializeField]
+        private string wrapT = null;
+
+        void ISerializationCallbackReceiver.OnAfterDeserialize()
+        {
+            if (Enum.TryParse(magFilter, out GltfMagnificationFilterMode result))
+            {
+                MagFilter = result;
+            }
+            else
+            {
+                MagFilter = GltfMagnificationFilterMode.Linear;
+            }
+            if (Enum.TryParse(minFilter, out GltfMinFilterMode result2))
+            {
+                MinFilter = result2;
+            }
+            else
+            {
+                MinFilter = GltfMinFilterMode.NearestMipmapLinear;
+            }
+            if (Enum.TryParse(wrapT, out GltfWrapMode result3))
+            {
+                WrapT = result3;
+            }
+            else
+            {
+                WrapT = GltfWrapMode.Repeat;
+            }
+            if (Enum.TryParse(wrapS, out GltfWrapMode result4))
+            {
+                WrapS = result4;
+            }
+            else
+            {
+                WrapS = GltfWrapMode.Repeat;
```

Comment:
In this class, you have correctly reset the value to the default specified by Khronos, but in other classes you haven't.  
Can we ensure the change is consistent and either preserves the default or does not.

Reply:
Can you point out cases where the default is dropped? I can't find any.